sdk/cognitiveservices/AnomalyDetector/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
-## 1.0.0-preview.1 (Unreleased)
+## 1.0.0 (2020-08-12)
+
+Changes in this release:
```

Comment:
Description should follow the pattern:
```
#### Breaking Changes
 - Breaking changes
                
#### New Features
 - New features            

#### Key Bug Fixes
 - Notable bug fixes
```
E.g.: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/identity/Azure.Identity/CHANGELOG.md